example/build.gradle

```diff
@@ -0,0 +1,46 @@
+apply plugin: 'com.android.library'
```

Comment:
There's nothing Android specific about this module. It exists only to host a Java class and Java test.